README.md

```diff
@@ -1,21 +1,22 @@
-# Fillr Senior JS Engineer Code Test
-
+# Fillr JS Engineer Code Test
 
 - Edit widget.js to create a JS 'widget' module that is executed on each frame. The widget should extract the names and labels for each html form control in the document. Each frame is already configured to load the widget script from widget/widget.js.
 
 - The top frame must collect the entire list of fields from all documents, including all descendent frames. The list of fields should be ordered by field 'name' in ascending order. See get_fields_test.js for the expected output.
 
-- When your widget has collected all the fields from all the frames, trigger a CustomEvent on the top frame document named 'frames:loaded' with a property named 'widget' within the detail. The widget is a string with the variable name of your widget object on the window scope.  This will call window[your_object_name].getFields() and run the test.
+- When your widget has collected all the fields from all the frames, trigger a CustomEvent on the top frame document named 'frames:loaded' with a property named 'fields' within the detail.
 
+- To run the test, make sure you have npm installed, and execute `./run/test`.
 
 Notes:
 
 - Only edit the widget.js file!
+- We highly encourage using postMessage to communicate between iframes if you're up for the challenge.
```

Comment:
Is it actually possible to solve the exercise _without_ using `postMessage`, in the case where the child frames are cross-domain?

If not, maybe it should be "You should use `postMessage` ...", rather than something that makes it sound optional.